src/tests/chain.test.ts

```diff
@@ -0,0 +1,28 @@
+import { getChainIdFromChain } from "../server/utils/chain";
```

Comment:
This test isn't working, due to ts compiler errors, don't have time to work through them so I manually tested this solution via a transaction

Reply:
In that case we should be able to delete this testing file - we don't have a formal test setup for engine right now so should be okay to keep this out.

Reply:
Awesome tysm